application/di/test_init.go

```diff
@@ -53,6 +54,8 @@ func TestInit(t *testing.T) {
 	defer initMutex.Unlock()
 	t.Helper()
 	c := config.CurrentConfig()
+	// we want to isolate CLI fake installs
+	c.CliSettings().SetPath(filepath.Join(t.TempDir(), "fake-cli"))
```

Comment:
Does this (and/or the server test changes below) fix the test flakes I've seen around the CLI downloads? :excited-conan-obrien-emoji:

Reply:
I hope so - had to add it at a few places.